hugo/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Shouldn't this inherit from the base file?

Reply:
There is no base file... I guess you mean the `core` package... I could refactor out a base file instead...